src/components/magasin/panel/index.js

```diff
@@ -193,7 +194,26 @@ export default class StorageUnitContainer extends Component {
     );
   }
 
+  getNodeTypeItems() {
```

Comment:
this implementation does not follow the jira issue description " (...) Dersom bruker velger en type som ikke er lovlig, må det gis melding om dette (tilsvarende som for andre valideringsfeil (...)".

In addition, I am not able to understand with a reasonable attempt what this method actually computes. I cannot understand what and why its doing.